### PR TITLE
Added Hindi translation.(Only Root READMe.md)

### PR DESCRIPTION
**Pull request by gaurang7goel**
Added Hindi language with README.hi.md name.
Only the root READMe.md for now.

### CODE REVIEW

**Review by matteobruni — CHANGES_REQUESTED** [review]
In issue #895 other users specified a folder for all the translations, please move the translated readme in that folder

**Comment by gaurang7goel** [comment]
ok sure

**Comment by gaurang7goel** [comment]
pls tell where to enter it , as i am unable to find  where to add.

**Comment by matteobruni** [comment]
https://github.com/matteobruni/tsparticles/issues/895#issuecomment-702174175

`Translation` folder like in #897

**Comment by gaurang7goel** [comment]
done pls check

File: traduction/README.hi.md [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+
+### **_NuGet_**
+
+[![Nuget](https://img.shields.io/nuget/v/tsParticles)](https://www.nuget.org/packages/tsParticles/)
+
+---
+
+### **_Usage_**
+
+Load tsParticles and configure the particles:
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+```html
+<div id="tsparticles"></div>
+
+<script src="tsparticles.min.js"></script>
+```
+
+**app.js**
+
+```javascript
+// @path-json can be an object or an array, the first will be loaded directly, the object from the array will be random selected
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+  .catch((error) => {
+    console.error(error);
+  });
+
+//or
+
+/* tsParticles.load(@dom-id, @options); */
+
+tsParticles.load("tsparticles", {
+  /* options here */
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+});
+
+//or
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 5]
```diff
@@ -0,0 +1,389 @@
...
+
+/* tsParticles.loadFromArray(@dom-id, @options, @index (optional)); */
+
+tsParticles.loadFromArray("tsparticles", [
+  {
+    /* options here */
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 6]
```diff
@@ -0,0 +1,389 @@
...
+  },
+  {
+    /* other options here */
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 7]
```diff
@@ -0,0 +1,389 @@
...
+  },
+]);
+//random object
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 8]
```diff
@@ -0,0 +1,389 @@
...
+    {
+      /* options here */
+    },
+    {
+      /* other options here */
+    },
+  ],
+  1
+); //the second one
+// Important! If the index is not in range 0...<array.length, the index will be ignored.
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 9]
```diff
@@ -0,0 +1,389 @@
...
+    /* other options here */
+  },
+]);
+//random object
+
+tsParticles.loadFromArray(
+  "tsparticles",
+  [
+    {
+      /* options here */
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 10]
```diff
@@ -0,0 +1,389 @@
...
+    },
+    {
+      /* other options here */
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 11]
```diff
@@ -0,0 +1,389 @@
...
+    },
+  ],
+  1
+); //the second one
+// Important! If the index is not in range 0...<array.length, the index will be ignored.
+
+// after initialization this can be used.
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 12]
```diff
@@ -0,0 +1,389 @@
...
+
+/* tsParticles.setOnClickHandler(@callback); */
+
+/* this will be fired from all particles loaded */
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 13]
```diff
@@ -0,0 +1,389 @@
...
+
+tsParticles.setOnClickHandler((event, particles) => {
+  /* custom on click handler */
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 14]
```diff
@@ -0,0 +1,389 @@
...
+});
+
+// now you can control the animations too, it's possible to pause and resume the animations
+// these methods don't change the config so you're safe with all your configurations
+// domItem(0) returns the first tsParticles instance loaded in the dom
+const particles = tsParticles.domItem(0);
+
+// play will start the animations, if the move is not enabled it won't enable it, it just updates the frame
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 15]
```diff
@@ -0,0 +1,389 @@
...
+
+/* this will be fired from all particles loaded */
+
+tsParticles.setOnClickHandler((event, particles) => {
+  /* custom on click handler */
+});
+
+// now you can control the animations too, it's possible to pause and resume the animations
+// these methods don't change the config so you're safe with all your configurations
+// domItem(0) returns the first tsParticles instance loaded in the dom
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 16]
```diff
@@ -0,0 +1,389 @@
...
+const particles = tsParticles.domItem(0);
+
+// play will start the animations, if the move is not enabled it won't enable it, it just updates the frame
+particles.play();
+
+// pause will stop the animations
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 17]
```diff
@@ -0,0 +1,389 @@
...
+
+## कुछ सबसे अधिक उपयोग किए जाने वाले ढांचे के लिए आधिकारिक घटक
+
+### Angular
+
+#### `ng-particles`
+
+[![npm](https://img.shields.io/npm/v/ng-particles)](https://www.npmjs.com/package/ng-particles) [![npm](https://img.shields.io/npm/dm/ng-particles)](https://www.npmjs.com/package/ng-particles)
+
+You can find the instructions [here](https://github.com/matteobruni/tsparticles/blob/master/components/angular/README.md)
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 18]
```diff
@@ -0,0 +1,389 @@
...
+
+### Inferno
+
+#### `inferno-particles`
+
+[![npm](https://img.shields.io/npm/v/inferno-particles)](https://www.npmjs.com/package/inferno-particles) [![npm](https://img.shields.io/npm/dm/inferno-particles)](https://www.npmjs.com/package/inferno-particles)
+
+You can find the instructions [here](https://github.com/matteobruni/tsparticles/blob/master/components/inferno/README.md)
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 19]
```diff
@@ -0,0 +1,389 @@
...
+
+### jQuery
+
+#### `jquery-particles`
+
+[![npm](https://img.shields.io/npm/v/jquery-particles)](https://www.npmjs.com/package/jquery-particles) [![npm](https://img.shields.io/npm/dm/jquery-particles)](https://www.npmjs.com/package/jquery-particles)
+
+You can find the instructions [here](https://github.com/matteobruni/tsparticles/blob/master/components/jquery/README.md)
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

File: traduction/README.hi.md [hunk 20]
```diff
@@ -0,0 +1,389 @@
...
+
+### Preact
+
+#### `preact-particles`
+
+[![npm](https://img.shields.io/npm/v/preact-particles)](https://www.npmjs.com/package/preact-particles) [![npm](https://img.shields.io/npm/dm/preact-particles)](https://www.npmjs.com/package/preact-particles)
+
+You can find the instructions [here](https://github.com/matteobruni/tsparticles/blob/master/components/preact/README.md)
```
Comment (matteobruni) [inline traduction/README.hi.md]: This is not translated

**Review by matteobruni — CHANGES_REQUESTED** [review]
Some titles are not translated too, double-check the file to see if you missed something in English

**Review by gaurang7goel — COMMENTED** [review]
Done , pls check once.

**Review by matteobruni — APPROVED** [review]
Looks good to me

**Comment by gaurang7goel** [comment]
Thanks,pls merge it .

**Review by matteobruni — APPROVED** [review]

**Comment by gaurang7goel** [comment]
thanks to check .Sorry for the mistakes I made, thanks to make them right

**Comment by gaurang7goel** [comment]
Pls tell if can help

**Comment by matteobruni** [comment]
Don't worry, mistakes can happen to everyone 😉

**Comment by matteobruni** [comment]
I need to wait automated checks to merge the PR

**Comment by gaurang7goel** [comment]
Thank you very much

**Comment by matteobruni** [comment]
Thank you for your contribution

**Pull request merged**